app/src/androidTest/java/com/github/fribourgsdp/radio/FirestoreDatabaseTest.kt

```diff
@@ -27,4 +27,14 @@ class FirestoreDatabaseTest {
         assertEquals( null,Tasks.await(user))
     }
 
+    @Test

```

Comment:
So far, we haven't tested this class, as since week 3, we avoid testing with API. If really we can keep this test, as we have sufficient API calls and google is rather a stable company, but in this case, set the id to a id that is not possibly generated by our application. If a user has Id 215 and this test run, troubles will occur.

Reply:
I've changed the test by creating a mock entry in our firestore DB, such that there is no risk the problems you stated may ever occur (the mock lobby entry has uid 123456789, whereas our lobby id's in game are always %1000).

Reply:
It is fine for me for me. It will likely disappear in the future as I have managed to find the way the mock the database so you're test will probably be replaced with a proper test that doesn't do any API call.